lib/releases.sh

```diff
@@ -38,15 +38,12 @@ if [ "${CAPM3RELEASEBRANCH}" == "release-0.5" ] || [ "${CAPM3_VERSION}" == "v1al
 elif [ "${CAPM3RELEASEBRANCH}" == "release-1.1" ]; then
   export CAPM3RELEASE="${CAPM3RELEASE:-$(get_latest_release "${CAPM3RELEASEPATH}" "v1.1.")}"
   export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v1.1.")}"
-else
+elif [ "${CAPM3RELEASEBRANCH}" == "release-1.2" ]; then
   export CAPM3RELEASE="${CAPM3RELEASE:-$(get_latest_release "${CAPM3RELEASEPATH}" "v1.2.")}"
   export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v1.2.")}"
-fi
-
-# Remove the following lines when CAPI/CAPM3 v1.3.0 releases are available
-if [ "${CAPM3RELEASEBRANCH}" == "main" ]; then
- export CAPIRELEASE="v1.3.0-beta.1"
- export CAPM3RELEASE="v1.3.0-beta.0"
```

Comment:
I would rather just keep this and not change anything else since this is more visible in the code what we are using.

Reply:
I did not want to change these each time a new pre-release is out from CAPM3/CAPI (rc.0, rc.1 from CAPM3 & rc.1 from CAPI) and fetch it automatically.

Reply:
I prefer this way rather. But ok for me if you dont want it this way.
/approve

Reply:
okay, I can fix it as before too and have to change some more things here.

/hold

Reply:
No leave it as you have done it in this PR now. I see your point since for the next pre release we have to come again and change it.

Reply:
Sure, left it as is

Reply:
/hold cancel